src/OpenFeature.Contrib.Hooks.Otel/README.md

```diff
@@ -64,6 +65,54 @@ In case something went wrong during a feature flag evaluation, you will see an e
 
 ![](./assets/otlp-error.png)
 
+## Usage - Metrics
+
+For this hook to function correctly a global `MeterProvider` must be set, an example of how to do this can be found below.
+
```

Comment:
Could you copy the table [here](https://github.com/open-feature/java-sdk-contrib/tree/main/hooks/open-telemetry#metricshook) and include it in this readme? I think we should actually add a section to the OpenFeature spec about this, but for now it's helpful to have them in the README I think.